src/pages/components/button/usage.mdx

```diff
@@ -329,46 +327,43 @@ Other fluid components include tiles and most recently, form inputs such as
 
 #### Fluid button border
 
-Many product designers have approached us looking for more guidance around
-borders between all fluid buttons. There is a 1px border between all fluid
-buttons that call the `$button-separator` token for borders. This feature adds a
-3:1 distinction between the two interactive UI elements. The border is a
-recommended feature to improve accessibility.
+There is [a 1px border](/components/button/style#ghost-button-structure.mdx)
```

Comment:
Is this link correct? Its going to ghost button structure.

Reply:
Instead of doing the inline link, maybe do the "See the style tab for details" type of thing at the end of the paragraph.